resources/lang/vi/community.php

```diff
@@ -22,108 +22,109 @@
     'support' => [
         'header' => [
             // size in font-size
-            'big_description' => 'Love osu!?<br/>
-                                Support the osu! dev :D',
+            'big_description' => 'Yêu thích osu!?<br/>
+                                Hãy ủng hộ các dev trong osu! :D',
             'small_description' => '',
-            'support_button' => 'I want to support osu!',
+            'support_button' => 'Tôi muốn ủng hộ osu!',
         ],
 
-        'dev_quote' => 'osu! is a completely free-to-play game, but running it is most definitely not so free. Between the cost of commissioning servers and high quality international bandwidth, the time spent maintaining the system and community, providing prizes for competitions, answering support questions and generally keeping people happy, osu! consumes quite a hefty amount of money! Oh, and don\'t forget the fact that we do it without any advertising or partnering with silly toolbars and the likes!
-            <br/><br/>osu! is at the end of the day largely run by myself, to which you may know best as "peppy".
-            I have had to quit my day job in order to keep up with osu!,
-            and do at times struggle to maintain the standards I strive for.
-            I would like to offer my personal thanks to those who have supported osu! thus far,
-            and just as much to those who continue to support this amazing game and community into the future :).',
+        'dev_quote' => 'osu! là một trò chơi hoàn toàn miễn phí, nhưng để cho osu chạy được thì lại không hề miễn phí. Giữa chi phí cho các máy chủ hoạt động và băng thông quốc tế chất lượng cao, thời gian dành ra để bảo trì hệ thống và cộng đồng osu!, việc cung cấp giải thưởng cho các cuộc thi, việc trả lời các câu hỏi giúp đỡ và giữ được sự vui vẻ cho mọi người như thường lệ, số lượng tiền mà osu tiêu tốn khá là khổng lồ. À, và chưa kể chúng tôi làm mà không nhờ đến sự quảng cáo nào hay hợp tác với nhưng thanh công cụ vớ vẫn và tương tự!
```

Comment:
- `nhưng để cho osu chạy được` thấy nó khá là... không hay lắm, nhưng mà không biết là phải dịch sao cho hợp, @ collaborators pls help
- `số lượng tiền mà osu tiêu tốn khá là khổng lồ.` bên kia có dấu chấm than mà :thonk:
- `À` tại sao không giữ nguyên `Ồ`?
- `nhưng` typo, must be `những`

Reply:
- fixed
- fixed
- fixed
- thực ra thì mình hiểu theo ý 'as usual'
- fixed with neutral tone

Reply:
`Neutral tone` là sao? Mình đọc thì thấy rằng thỉnh thoảng đám dev cũng tự mỉa mai bản thân mình mà.